### PR TITLE
fix(*): kongponents alpha phase 15 [KHCP-12191]

### PR DESCRIPTION
**Pull request by portikM**
# Summary

Kongponents alpha phase 15 adoption. Includes breaking changes from https://github.com/Kong/kongponents/pull/2204.

#### Resources

- [Creating a package docs](https://github.com/Kong/public-ui-components/blob/main/docs/creating-a-package.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
     "@evilmartians/lefthook": "^1.6.14",
     "@kong/design-tokens": "1.13.2",
     "@kong/eslint-config-kong-ui": "^1.0.3",
-    "@kong/kongponents": "9.0.0-alpha.162",
+    "@kong/kongponents": "9.0.0-alpha.166",
+    "@rushstack/eslint-patch": "^1.7.2",
```
Comment (adamdehaven) [inline package.json:30]: issue: bad rebase

**Review by adamdehaven — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ValeryG) [inline package.json:30]: what's this one for?

File: packages/entities/entities-consumer-groups/src/components/ConsumerGroupList.cy.ts [hunk 2]
```diff
@@ -310,10 +310,10 @@ describe('<ConsumerGroupList />', () => {
         props: {
           cacheIdentifier,
           config: baseConfigKM,
-          canCreate: () => {},
-          canEdit: () => {},
-          canDelete: () => {},
-          canRetrieve: () => {},
+          canCreate: () => { },
```
Comment (ValeryG) [inline packages/entities/entities-consumer-groups/src/components/ConsumerGroupList.cy.ts:313]: are those changes coming from lint rules ?

File: package.json (hunk 1 above)
Comment (portikM) [reply package.json]: Sorry, bad rebase, fixed now

**Review by adamdehaven — DISMISSED** [review]

File: packages/entities/entities-consumer-groups/src/components/ConsumerGroupList.cy.ts (hunk 2 above)
Comment (portikM) [reply packages/entities/entities-consumer-groups/src/components/ConsumerGroupList.cy.ts]: Eh no, my VSC formatted them this way for some reason but I backed out these formatting changes now

**Review by adamdehaven — APPROVED** [review]

File: packages/entities/entities-consumer-groups/src/components/ConsumerGroupList.cy.ts (hunk 2 above)
Comment (portikM) [reply packages/entities/entities-consumer-groups/src/components/ConsumerGroupList.cy.ts:313]: I take that back, those are the lint rules

**Pull request merged**